imap_processing/mag/mag_cdf_attrs.py

```diff
@@ -44,15 +44,15 @@
 
 
 mag_l1b_attrs = GlobalDataLevelAttrs(
-    "L1A>Level-1B",
+    "L1B_SCI>Level-1B Science Data",
     # TODO: replace "sci" with descriptor "norm" / "burst"
     logical_source="imap_mag_l1b_sci",
     logical_source_desc="IMAP Mission MAG Instrument Level-1B Data.",
     instrument_base=mag_base,
 )
 
 mag_l1c_attrs = GlobalDataLevelAttrs(
-    "L1A>Level-1C",
+    "L1A_SCI>Level-1C Science Data",
```

Comment:
Should "L1A_SCI" be "L1C_SCI" for the l1c attrs? Just looking at l1b attrs for comparison, but I could be missing something important.

Reply:
good catch. Yes. it should be.